#!/usr/bin/env python
PACKAGE = "euclid_arduino_wrapper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("depth_roi_x",    int_t,    0, "X value of the left upper corner", 0,  0, 320)
gen.add("depth_roi_y",    int_t,    0, "Y value of the left upper corner", 0,  0, 240)
gen.add("depth_roi_width",    int_t,    0, "ROI width", 320,  1, 320)
gen.add("depth_roi_height",    int_t,    0, "ROI height", 240,  1, 240)






exit(gen.generate(PACKAGE, "euclid_arduino_wrapper", "EuclidArduinoWrapper"))
